cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.tsx

```diff
@@ -0,0 +1,172 @@
+// React imports
+import React, { ChangeEvent, useState } from 'react';
+
+// Library imports
+import { ClarityIcons, blockIcon, blocksGroupIcon, clusterIcon } from '@cds/core/icon';
+import { CdsControlMessage, CdsFormGroup } from '@cds/react/forms';
+import { CdsInput } from '@cds/react/input';
+import { useForm } from 'react-hook-form';
+import { CdsRadio, CdsRadioGroup } from '@cds/react/radio';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsButton } from '@cds/react/button';
+
+// App imports
+import { INPUT_CHANGE } from '../../../../state-management/actions/Form.actions';
+import { StepProps } from '../../../../shared/components/wizard/Wizard';
+
+ClarityIcons.addIcons(blockIcon, blocksGroupIcon, clusterIcon);
+
+interface FormInputs {
+    CLUSTER_NAME: string;
+    CLUSTER_SERVICE_CIDR: string;
+    CLUSTER_POD_CIDR: string;
+}
+
+const unmanagedClusterProviders = [
+    {
+        label: 'calico',
+        value: 'CALICO',
+    },
+    {
+        label: 'anthrea',
+        value: 'ANTHREA',
+    },
+    {
+        label: 'none',
+        value: 'NONE',
+    },
+];
+function UnmanagedClusterNetworkSettings(props: Partial<StepProps>) {
+    const { handleValueChange, currentStep } = props;
+
+    const {
+        register,
+        formState: { errors },
+    } = useForm<FormInputs>();
+
+    const handleClusterNameChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Since the field name is available as event.target.name, you could have a single handler (rather than three)

Reply:
+1 also, for consistency you can define a constants file that includes the field names instead of using strings like 'CLUSTER_NAME.' You can see an example of this in Shimon's PR at https://github.com/vmware-tanzu/community-edition/pull/4749/files#diff-4c902f152fa27e32ba85bf51aa88d85822b80bd5c11bb3e027f1f26fd1092389

and then how he consumes them in

https://github.com/vmware-tanzu/community-edition/pull/4749/files#diff-6d54e752cbe370988f5eae8f462ac9f7918f679eafe0a76df2f3338c8bb93bb4